src/homepageExperience/components/OptionAccordion/QueryDataAccordion.tsx

```diff
@@ -46,6 +46,8 @@ export const QueryDataAccordion: FC = () => {
   const languageList = {
```

Comment:
Let's list these alphabetically like the "Write Data" list. Also, it is getting cut off, it should scroll like the other one. 
![image](https://github.com/influxdata/ui/assets/11937365/6e1bd89a-d40a-4b04-aa2f-24b0f4ed1bed)

Reply:
The list is sorted alphabetically.

I'm unable to modify the scss to display the entire list. For the Add Data dropdown, it works because there's sufficient space beneath it:

![image](https://github.com/influxdata/ui/assets/455137/dd340a65-4f94-422c-a72c-2b369e251798)

A quick but inelegant solution would be to set the max-height to 75px. However, this seems like an unsightly workaround to me.

Reply:
Found a fix fo the dropdown issue. In the `OptionAccordion.scss` file, make this change:
<img width="350" alt="image" src="https://github.com/influxdata/ui/assets/11937365/930655b9-f0b2-426b-89c4-0bafe3b377f9">

This will allow the section with the dropdown to appear above the others like this:
![image](https://github.com/influxdata/ui/assets/11937365/fdc7901c-a025-4f9c-a83c-228728f5f495)